inst/EpiModel2.rmd

```diff
@@ -56,13 +63,13 @@ target.stats <- c(30, 0)
 
 It is important to note, that the vertex attribute "group" now takes on a special meaning within the context of EpiModel modeling, separate from other attribute names. Users should be mindful when constructing networks in which "group" is a vertex attribute and note whether or not the interpretation of group aligns with that detailed above.
 
-Further, the group vertex attribute is restricted to levels of 1 and 2; other combinations of attribute levels - 0 and 1 or "male" and "female" - are not permitted here and will result in unexpected behavior in EpiModel. Further note that we are not limited to replicating bipartite networks and the target statistic for `nodematch("group")` can be any number from 0 to the total number of edges in the network.
+Further, the group vertex attribute is restricted to levels of 1 and 2; other combinations of attribute levels - say, 0 and 1 or "male" and "female" - are not permitted here and may result in EpiModel producing unintended results. Finally, we are not limited to replicating bipartite networks and the target statistic for `nodematch("group")` can be any number from 0 to the total number of edges in the network.
```

Comment:
Before mentioned that the target statistic can be any number from 0 to total edges in the network, is the part about not being limited to replicated bipartite networks necessary? Seems like that is the whole point of all that you've written above.

```suggestion
Further, the group vertex attribute is restricted to levels of 1 and 2. Other combinations of attribute levels (e.g., 0 and 1 or "male" and "female") are not permitted here and may result in EpiModel producing unintended results. Finally, we are not limited to replicating bipartite networks and the target statistic for `nodematch("group")` can be any number from 0 to the total number of edges in the network.
```